scripts/iso-manager.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+BOOTSTRAP_CONFIG="/etc/metalk8s/bootstrap.yaml"
+DEBUG="info"
+LOGFILE="/tmp/iso.log"
+DRY_RUN="False"
+
+# Helpers
+_usage() {
+echo "iso-manager.sh [options]"
+echo "Options:"
+echo "-p/--product:     Path to product folder or ISO"
+echo "-d/--dry-run:     Run actions in dry run mode"
+echo "-l/--log-file:    path to log file"
```

Comment:
To be consistent use `verbose` instead of `DEBUG` and add an option to change it